rdb/src/main/java/com/instana/dc/rdb/DbRawMetricRegistry.java

```diff
@@ -61,6 +61,10 @@ public class DbRawMetricRegistry {
         put(DB_USER_OVERFLOW_NAME, new RawMetric(UPDOWN_COUNTER, DB_USER_OVERFLOW_NAME, DB_USER_OVERFLOW_DESC, DB_USER_OVERFLOW_UNIT, true, null));
         put(DB_SEQ_SCAN_NAME, new RawMetric(UPDOWN_COUNTER, DB_SEQ_SCAN_NAME, DB_SEQ_SCAN_DESC, UNIT_BY, true, DB_SEQ_SCAN_KEY));
         put(DB_SEQ_SCAN_TABLE_NAME, new RawMetric(GAUGE, DB_SEQ_SCAN_TABLE_NAME, DB_SEQ_SCAN_TABLE_DESC, DB_SEQ_SCAN_TABLE_UNIT, true, null));
+        put(DB_LOCK_WAITS_NAME, new RawMetric(UPDOWN_COUNTER, DB_LOCK_WAITS_NAME, DB_LOCK_WAITS_DESC, DB_LOCK_WAITS_UNIT, true, null));
+        put(DB_CACHE_READ_RATIO_NAME, new RawMetric(UPDOWN_COUNTER, DB_CACHE_READ_RATIO_NAME, DB_CACHE_READ_RATIO_DESC, DB_CACHE_READ_RATIO_UNIT, true, null));
```

Comment:
`DB_CACHE_WRITE_RATIO_NAME ` & `DB_CACHE_READ_RATIO_NAME `  instrumentation should be `Gauge` not  `UPDOWN_COUNTER` .